lms/djangoapps/learner_dashboard/README.rst

```diff
@@ -0,0 +1,22 @@
+# Learner Dashboard
```

Comment:
@schenedx we're trying to standardize on RST. READMEs in the codebase that used to be Markdown have all been converted to RST.